examples/plot_nonstatconvolve.py

```diff
@@ -79,15 +78,15 @@
 # filter
 nx, nz = 601, 501
 
-wav1a, _, wav1c = ricker(t[:17], f0=12)
-wav1b = ricker(t[:17], f0=30)[0]
+wav1a, _, _ = ricker(t[:17], f0=12)
+wav1b, _, _ = ricker(t[:17], f0=30)
 wav2a = gaussian(35, 2.0)
 wav2b = gaussian(35, 4.0)
 
 wav11 = np.outer(wav1a, wav2a[np.newaxis]).T
 wav12 = np.outer(wav1b, wav2a[np.newaxis]).T
 wav21 = np.outer(wav1b, wav2b[np.newaxis]).T
-wav22 = np.outer(wav1b, wav2b[np.newaxis]).T
+wav22 = np.outer(wav1a, wav2b[np.newaxis]).T
```

Comment:
Without this change, `wav21` and `wav22` are the same.

Reply:
Makes sense :)